flink-cdc-runtime/src/main/java/com/ververica/cdc/runtime/serializer/schema/ColumnWithPositionSerializer.java

```diff
@@ -78,23 +82,23 @@ public int getLength() {
     @Override
     public void serialize(AddColumnEvent.ColumnWithPosition record, DataOutputView target)
             throws IOException {
-        columnSerializer.serialize(record.getAddColumn(), target);
+        addColumnSerializer.serialize(record.getAddColumn(), target);
         positionEnumSerializer.serialize(record.getPosition(), target);
-        if (record.getExistingColumn() == null) {
+        if (record.getExistedColumnName() == null) {
```

Comment:
This condition code is useless. `StringSerializer` will help to deal with `null`.

Change to:
```java
addColumnSerializer.serialize(record.getAddColumn(), target);
positionEnumSerializer.serialize(record.getPosition(), target);
stringSerializer.serializer(record.getExistedColumnName(), target);
```